crypto/ecdsa/ecdsa.go

```diff
@@ -0,0 +1,132 @@
+package neofsecdsa
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/sha512"
+	"errors"
+	"math/big"
+	"sync"
+
+	neofscrypto "github.com/nspcc-dev/neofs-api-go/crypto"
+)
+
+type ecdsaSigner ecdsa.PrivateKey
+
+var (
+	onceCurve sync.Once
```

Comment:
can't it be done with `init()` func here and beyong?

Reply:
It can be, but not all public functionality requires this initialization, so in special cases it will be redundant.